tests/utils/test_verification_utils.py

```diff
@@ -20,13 +21,28 @@ def test_recover_relative_pose_from_essential_matrix(self):
         corr_i1, corr_i2, i2Ei1 = simulate_two_planes_scene(10, 10)
 
         i2Ri1, i2Ui1 = verification_utils.recover_relative_pose_from_essential_matrix(
-            i2Ei1.matrix(),
-            corr_i1.coordinates,
-            corr_i2.coordinates,
-            Cal3Bundler(),
-            Cal3Bundler(),
+            i2Ei1.matrix(), corr_i1.coordinates, corr_i2.coordinates, Cal3Bundler(), Cal3Bundler()
         )
 
         # compare the recovered R and U with the ground truth
         self.assertTrue(i2Ri1.equals(i2Ei1.rotation(), 1e-3))
         self.assertTrue(i2Ui1.equals(i2Ei1.direction(), 1e-3))
+
+    def test_compute_epipolar_distances(self):
+        """Test for epipolar distance computation using 2 sets of points and the essential matrix."""
```

Comment:
where does this example come from? maybe we could add some more intuition behind it

Reply:
I just wrote some random points. No intuition behind them.